packages/paste-core/primitives/listbox/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,166 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import {useListboxPrimitiveState, ListboxPrimitive, ListboxPrimitiveItem} from '../src';
+
+const FIRST_ITEM = 'first-listbox-item';
+const SECOND_ITEM = 'second-listbox-item';
+
+const ListboxMock: React.FC = () => {
+  const [selected, setSelected] = React.useState<number>();
+  const listbox = useListboxPrimitiveState({baseId: 'listbox-example'});
+  return (
+    <ListboxPrimitive {...listbox} aria-label="Code">
+      <ListboxPrimitiveItem
+        {...listbox}
+        data-testid="first-listbox-item"
```

Comment:
question: why are you using strings to define the id's here rather than your constants?

Reply:
Good catch! Forgot to refactor those bits 😛